tests/translator/input/simple_function_with_tags_false.yaml

```diff
@@ -0,0 +1,24 @@
+Resources:
+  MyLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs18.x
+      InlineCode: |
+        exports.handler = async (event, context, callback) => {
+          return {
+            statusCode: 200,
+            body: 'Success'
+          }
+        }
+      MemorySize: 128
+      Policies:
+      - AWSLambdaRole
+      - AmazonS3ReadOnlyAccess
+      Tags:
+        TagKey1: TagValue1
+        TagKey2: ''
+        TagKey3: false
+        TagKey4: true
```

Comment:
`TagKey5: 0` missing here?

Reply:
Globals and in function seem to tag using the same function.